factory/defaults/chains/testnet.yaml

```diff
@@ -120,6 +120,12 @@ chains:
     external:
       indexing_co:
         chain_id: eth_holesky
+  FIL:
+    chain: FIL
+    driver: fil
+    chain_name: fil
```

Comment:
can we use (renaming driver to filecoin):

```
    driver: filecoin 
    chain_name: Filecoin
```

I think it's a little more consistent